src/main/java/com/bakdata/kafka/KafkaConnectorSourceResetter.java

```diff
@@ -188,4 +189,13 @@ private Consumer<byte[], byte[]> createConsumer(final Map<String, Object> kafkaC
         return consumer;
     }
 
+    private <K, V> List<PartitionInfo> partitionsForOffsetTopic(final Consumer<K, V> consumer) {
+        final Map<String, List<PartitionInfo>> topicsWithPartition = consumer.listTopics();
+        if (!topicsWithPartition.containsKey(this.offsetTopic)) {
+            final String message = String.format("Topic %s does not exist.", this.offsetTopic);
```

Comment:
```suggestion
            final String message = String.format("Topic '%s' does not exist.", this.offsetTopic);
```
Helps to see accidental white space